docs/references/resolvers.md

```diff
@@ -45,3 +45,46 @@ class Color:
             return list(self.fruits.objects.filter(...))
         return query()
 ```
+
+## Issues with Resolvers
+
+It is important that if you override resolvers, you will lose access to all of the things that come by default
+with the library (e.g. `Pagination`, `Filter`). For Types that are attached to your base `Query`, it
+is recommended to define a custom `get_queryset` on the Type
```

Comment:
I wouldn't say that it is recommended. I think that it is important to explain that for root fields (the ones defined on `Query`), `get_queryset` can be used.

But I would add a warning there, because it might confuse people into thinking that it would always be used like that, and it would affect things like retrieving related fields.